nipype/interfaces/niftyreg/base.py

```diff
@@ -105,8 +105,11 @@ def exists(self):
     def _run_interface(self, runtime):
         # Update num threads estimate from OMP_NUM_THREADS env var
         # Default to 1 if not set
-        if not isdefined(self.inputs.environ['OMP_NUM_THREADS']):
-            self.inputs.environ['OMP_NUM_THREADS'] = self.num_threads
+        omp_key = 'OMP_NUM_THREADS'
+        if omp_key in os.environ:
```

Comment:
I'd suggest this instead the `if .. else`:
`self.inputs.environ[omp_key] = os.getenv(omp_key, '%d' % self.num_threads)`